entities/TestConfiguration.js

```diff
@@ -121,18 +151,114 @@ class TestConfiguration {
       case 'equal_to':
         result = responseHeaders[assertionHeader.property] === assertionHeader.target;
         break;
-        case 'contains':
-          result = responseHeaders[assertionHeader.property].includes(assertionHeader.target);
-          break;
-        case 'not_contains':
-          result = !responseHeaders[assertionHeader.property].includes(assertionHeader.target); 
-          break;
-        case 'greater_than_or_equal_to': 
-          result = Number(responseHeaders[assertionHeader.property]) >= Number(assertionHeader.target);
-          break;
-        case 'less_than_or_equal_to':
-          result = Number(responseHeaders[header.property]) <= Number(assertionHeader.target); 
-          break 
+      case 'contains':
+        result = responseHeaders[assertionHeader.property].includes(assertionHeader.target);
+        break;
+      case 'not_contains':
+        result = !responseHeaders[assertionHeader.property].includes(assertionHeader.target);
+        break;
+      case 'greater_than_or_equal_to':
+        result = Number(responseHeaders[assertionHeader.property])
+        >= Number(assertionHeader.target);
+        break;
+      case 'less_than_or_equal_to':
+        result = Number(responseHeaders[assertionHeader.property])
+        <= Number(assertionHeader.target);
+        break;
+      default:
+        result = false;
+    }
+    return result;
+  }
+
+  static checkJsonBody(targetValue, actualValue, comparisonType) {
```

Comment:
Wondering if we could combine some of the `checkJsonBody` and `checkHeaders` code? Not crucial for v1 but maybe we should add as a TODO?